app.py

```diff
@@ -37,43 +37,64 @@ def init_app(self):
             # provide any commands!
             return
 
-        # register the file open command
-        self.engine.register_command(
-            "File Open...",
-            self.show_file_open_dlg,
-            {
-                "short_name": "file_open",
-                # dark themed icon for engines that recognize this format
-                "icons": {
-                    "dark": {
-                        "png": os.path.join(
-                            os.path.dirname(__file__),
-                            "resources",
-                            "file_open_menu_icon.png",
-                        )
-                    }
+        if self.get_setting("context_change_only_mode"):
```

Comment:
I think a more usefull approach for clients would be to have the following setting and their defaults

```
show_file_open: true
show_file_save: true
shot_change_context: false
```

This way we allow clients to select what they want. What do you think?

Reply:
I don't really, mind, I guess it allows more flexibly. I'm not sure when I would have the open app and not the save app, I guess maybe in a project context